app/src/main/res/layout/act_result_save.xml

```diff
@@ -155,6 +155,7 @@
                 android:importantForAutofill="no"
                 android:minHeight="48dp"
                 android:textAlignment="textEnd"
+                android:inputType="none"
```

Comment:
Add a to-do to correct the keyboard bug (right now does not display a : and we set that on the input digits).